### PR TITLE
#3813 Disable action buttons when a Paper element is in the pending state

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3813)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/PipelineApiHelper/stream.js [hunk 1]
```diff
@@ -104,7 +104,9 @@ const stream = () => {
     paperApi.updateElement(id, {
       status: CELL_STATUS.pending,
     });
+    paperApi.enableMenu(id);
     const res = await startStream(name);
+    paperApi.disableMenu(id);
```
Comment (eechih) [inline ohara-manager/client/src/components/Pipeline/PipelineApiHelper/stream.js]: enableMenu 與 disableMenu 順序好像相反了

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/PipelineApiHelper/stream.js]: Yep, my bad. Thanks :)

File: ohara-manager/client/src/components/Pipeline/Paper/Paper.js [hunk 2]
```diff
@@ -616,13 +613,21 @@ const Paper = React.forwardRef((props, ref) => {
...
-          ({ model }) => model.get('id') === id || model.get('name') === id,
-        );
+        const cellView = findCellView(id);
 
         cellView && cellView.highlight();
       },
 
+      enableMenu(id, items) {
+        const cellView = findCellView(id);
+        cellView && cellView.enableMenu(items);
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Paper/Paper.js]: @wu87988622 new methods for enabling and disabling an element's menu

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**